docs/how_to/query.md

```diff
@@ -0,0 +1,47 @@
+# Setting up m3query
+
+## Introduction
+
+m3query is used to query data that is stored in M3DB. For instance, if you are using the Prometheus remote write endpoint with [m3coordinator](../integrations/prometheus.md), you can use m3query instead of the Prometheus remote read endpoint. By doing so, you get all of the benefits of m3query's engine. Furthermore, since m3query provides a Prometheus compatible API, you use 3rd party graphing and alerting solutions like Grafana.
```

Comment:
Do we capitalize? I think we capitalize M3DB usually but not sure about query.

Reply:
I haven't been capitalizing `m3query`